packages/cadl-rlc-test/cadl-output/src/parameters.ts

```diff
@@ -0,0 +1,12 @@
+import { RequestParameters } from "@azure-rest/core-client";
+import { Widget } from "./models";
+
+export type WidgetServiceListParameters = RequestParameters;
+export type WidgetServiceReadParameters = RequestParameters;
+
+export interface WidgetServiceCreateBodyParam {
+    body: Widget;
+}
+
+export type WidgetServiceCreateParameters = WidgetServiceCreateBodyParam & RequestParameters;
+export type WidgetServiceCustomGetParameters = RequestParameters;
```

Comment:
Compare with the file autorest generated https://github.com/Azure/autorest.typescript/pull/1471/files#diff-ab653ac0d8bb20bf748634d4ea6e4debc829adfae78ee1ee525eb8bbac25e6ff. Seems we have extra prefix of WidgetService of each parameters.  Also, there's some parameters definition missing, like CreateMediaTypeParam and ReadParameters.

Reply:
@qiaozha Thanks for pointing this out!

I will fix the extra prefix issue here. The ReadParameters is not missing one and we have `WidgetServiceReadParameters`.

But the `CreateMediaTypeParam` is missing in our case. This is introduced by the swagger code(https://github.com/Azure/autorest.typescript/blob/2e9066f7870e06032ac19626b2ba7a0d6d7c16ca/packages/autorest.typescript/test/smoke/swagger/widget.json#L46) generated from CADL. And I think our CADL one is correct.

Reply:
I'm thinking that we may always want to add the prefix regardless of how many operation groups we have. The reason is to avoid breaking changes if the swagger/cadl adds a new operation group in the future. With the current design, the old Parameters and Responses will now have a Prefix (because now there are 2 operation groups). This will be a breaking change. This may be a good opportunity to change this

Reply:
Good point Jose. Do you think we need to adjust autorest.typescript this time?

Reply:
Should we also change the name in autorest typescript to remove this ?

Reply:
updated